src/test/java/org/opensearch/geospatial/ip2geo/common/GeoIpDataFacadeTests.java

```diff
@@ -52,6 +52,7 @@
 import org.opensearch.action.search.SearchResponse;
 import org.opensearch.action.support.IndicesOptions;
 import org.opensearch.common.Randomness;
+import org.opensearch.common.Strings;
```

Comment:
please make sure if works with latest update from core, they start moving classes around https://github.com/opensearch-project/OpenSearch/pull/7508

Reply:
Yes. There is another PR to fix it.https://github.com/opensearch-project/geospatial/pull/314